### ISSUE TITLE
Make markers stack in minimap view

### ISSUE THREAD

**Issue by johnkslg** [StrangeLoopGames/EcoIssues#12522]
![image](https://user-images.githubusercontent.com/3536496/59314540-92461800-8c6a-11e9-8690-749f1fae9666.png)

**Comment by beastyn** [comment]
@johnkslg what does it mean exactly? Do we still beed this?

**Comment by IcedForge** [comment]
Yes please, would still like to see this happen, BUT i would also like either a feature option to turn off automatically creating markers for crafting - as when you start to have a lot of orders rotating they clog up the markers list in the map view and delete all would also delete any user manual made markers.
![Screenshot_414](https://user-images.githubusercontent.com/24981362/89735896-e662ad00-da65-11ea-8c73-f2d9b8d33421.jpg)
